tfjs-backend-cpu/src/kernels/LRN.ts

```diff
@@ -59,8 +59,9 @@ export function lRN(
   return backend.makeTensorInfo(x.shape, x.dtype, result);
 }
 
-export const lRNConfig: KernelConfig = {
+// tslint:disable-next-line: variable-name
+export const LRNConfig: KernelConfig = {
   kernelName: LRN,
   backendName: 'cpu',
-  kernelFunc: lRN as {} as KernelFunc
+  kernelFunc: lrn as {} as KernelFunc
```

Comment:
nit: Should here use `lRN` instead of `lrn` since you use `LRNConfig`?

Reply:
WebGL defines this as lrn, so I just keep them consistent (node doesn't have a specific name for this).

Reply:
To be consistent with LRNGrad, I will revert the change here.